### PR TITLE
OCP 4.14 Release Notes for P/Z

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s): 4.14

Issue:
https://issues.redhat.com/browse/MULTIARCH-3706

Draft for P/Z :
https://docs.google.com/document/d/145VCLtoOxZ58hgYhqmDohgbm6DPs5VsMyeg8q_DAE9Y/edit#heading=h.kjy2bqn80j2x

Link to docs preview:
P: https://64616--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-ibm-power
Z: https://64616--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-ibm-z

QE review:
For P : Manoj Kumar, Luvella , Julie
for Z : Holger Wolf, Florian Leber

Additional information:

Branch : enterprise-4.14
@SNiemann15

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -158,9 +158,352 @@ For more information, see xref:../applications/creating_applications/creating-ap
...
+|Helm
+|Supported
+|Supported
+
+|Horizontal pod autoscaling
+|Supported
+|Supported
+
+|IBM Secure Execution
+|Supported
```
Comment (SNiemann15) [inline release_notes/ocp-4-14-release-notes.adoc:279]:
```suggestion
|Unsupported
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -158,9 +158,352 @@ For more information, see xref:../applications/creating_applications/creating-ap
...
+
+This release introduces support for the following features on {ibmzProductName} and {linuxoneProductName}:
+
+* Assisted Installer with z/VM
+* Installing on a single node
+* Hosted control planes (Technology Preview)
+* Multi-architecture compute nodes
+* oc-mirror plugin
+
+[discrete]
```
Comment (SNiemann15) [inline release_notes/ocp-4-14-release-notes.adoc:186]: @holgwolf Shall we keep this entry from the 4.13 RNs or is it sufficient to have SE in the supported features table.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -158,9 +158,352 @@ For more information, see xref:../applications/creating_applications/creating-ap
...
+* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc#installing-restricted-networks-ibm-z-kvm[Installing a cluster with RHEL KVM on {ibmzProductName} and {linuxoneProductName} in a restricted network]
+
+[IMPORTANT]
+====
+Compute nodes must run {op-system-first}
+====
+
+[discrete]
+==== {ibmzProductName} and {linuxoneProductName} notable enhancements
+
```
Comment (SNiemann15) [inline release_notes/ocp-4-14-release-notes.adoc:175]:
```suggestion

Starting with {product-title} 4.14, Extended Update Support (EUS) is extended to the {ibmzProductName} platform.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 4]
```diff
@@ -158,9 +158,352 @@ For more information, see xref:../applications/creating_applications/creating-ap
...
+* xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on {ibmpowerProductName} in a restricted network]
+
+[IMPORTANT]
+====
+Compute nodes must run {op-system-first}
+====
+
+[discrete]
+==== {ibmpowerProductName} notable enhancements
+
```
Comment (SNiemann15) [inline release_notes/ocp-4-14-release-notes.adoc:210]:
```suggestion

Starting with {product-title} 4.14, Extended Update Support (EUS) is extended to the {ibmpowerProductName} platform.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].

```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 5]
```diff
@@ -158,9 +158,356 @@ For more information, see xref:../applications/creating_applications/creating-ap
...
+Starting with {product-title} 4.14, Extended Update Support (EUS) is extended to the {ibmzProductName} platform.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
+
+The {ibmzProductName} and {linuxoneProductName} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.
+
+This release introduces support for the following features on {ibmzProductName} and {linuxoneProductName}:
+
+* Assisted Installer with z/VM
+* Installing on a single node
+* Hosted control planes (Technology Preview)
```
Comment (SNiemann15) [inline release_notes/ocp-4-14-release-notes.adoc:184]:
```suggestion
// * Hosted control planes (Technology Preview)
```
Needs to be added through CM after 4.14 GA see discussion in this [thread](https://redhat-internal.slack.com/archives/CFFJUNP6C/p1694770713321609).

File: release_notes/ocp-4-14-release-notes.adoc [hunk 6]
```diff
@@ -158,9 +158,356 @@ For more information, see xref:../applications/creating_applications/creating-ap
...
+Starting with {product-title} 4.14, Extended Update Support (EUS) is extended to the {ibmpowerProductName} platform.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
+
+The {ibmpowerProductName} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.
+
+This release introduces support for the following features on {ibmpowerProductName}:
+
+* {ibmpowerProductName} Virtual Server Block CSI Driver Operator (Technology Preview)
+* Installer-provisioned Infrastructure Enablement for {ibmpowerProductName} Virtual Server (Technology Preview)
+* Installing on a single node
+* Hosted control planes (Technology Preview)
```
Comment (SNiemann15) [inline release_notes/ocp-4-14-release-notes.adoc:222]:
```suggestion
// * Hosted control planes (Technology Preview)
```
Needs to be added through CM after 4.14 GA see discussion in this [thread](https://redhat-internal.slack.com/archives/CFFJUNP6C/p1694770713321609).

File: release_notes/ocp-4-14-release-notes.adoc [hunk 7]
```diff
@@ -158,9 +158,356 @@ For more information, see xref:../applications/creating_applications/creating-ap
...
+|Supported
+|Supported
+
+|File Integrity Operator
+|Supported
+|Supported
+
+|HyperShift Operator
+|Technology Preview
+|Technology Preview
```
Comment (SNiemann15) [inline release_notes/ocp-4-14-release-notes.adoc:440]:
```suggestion
// |HyperShift Operator
// |Technology Preview
// |Technology Preview
```
Needs to be added through CM after 4.14 GA see discussion in this [thread](https://redhat-internal.slack.com/archives/CFFJUNP6C/p1694770713321609).

File: release_notes/ocp-4-14-release-notes.adoc (hunk 5 above)
Comment (alishaIBM) [reply release_notes/ocp-4-14-release-notes.adoc]: Updated the PR.

File: release_notes/ocp-4-14-release-notes.adoc (hunk 6 above)
Comment (alishaIBM) [reply release_notes/ocp-4-14-release-notes.adoc]: Updated the PR.

File: release_notes/ocp-4-14-release-notes.adoc (hunk 7 above)
Comment (alishaIBM) [reply release_notes/ocp-4-14-release-notes.adoc]: Updated the PR.

**Comment by manojnkumar** [comment]
Under the `Known Issues` section we need a new paragraph for the kernel crash.
https://64616--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-known-issues

When you deploy a cluster with a Network Type of OVNKubernetes on IBM Power, compute nodes may reboot because of a kernel stack overflow.  

Workaround: Deploy the cluster with a Network Type of OpenShiftSDN. [https://issues.redhat.com/browse/RHEL-3901](url)

**Comment by alishaIBM** [comment]
> Under the `Known Issues` section we need a new paragraph for the kernel crash. https://64616--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-known-issues
> 
> When you deploy a cluster with a Network Type of OVNKubernetes on IBM Power, compute nodes may reboot because of a kernel stack overflow.
> 
> Workaround: Deploy the cluster with a Network Type of OpenShiftSDN. [https://issues.redhat.com/browse/RHEL-3901](url)

Addressed in a separate PR: [OCP 4.14 RN's: Update known issues](https://github.com/openshift/openshift-docs/pull/65853) fyi.

**Comment by SNiemann15** [comment]
/lgtm 

@holgwolf Can you please QE review and approve for IBM Z
Also please see my comment [above](https://github.com/openshift/openshift-docs/pull/64616#discussion_r1323225967).

**Comment by holgwolf** [comment]
LGTM

**Comment by alishaIBM** [comment]
@manojnkumar can you please QE review and approve for IBM Power

/assign @manojnkumar

File: release_notes/ocp-4-14-release-notes.adoc [hunk 8]
```diff
@@ -203,9 +203,356 @@ For more information, see xref:../applications/creating_applications/creating-ap
...
+==== {ibmpowerProductName} notable enhancements
+
+Starting with {product-title} 4.14, Extended Update Support (EUS) is extended to the {ibmpowerProductName} platform.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
+
+The {ibmpowerProductName} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.
+
+This release introduces support for the following features on {ibmpowerProductName}:
+
+* {ibmpowerProductName} Virtual Server Block CSI Driver Operator (Technology Preview)
+* Installer-provisioned Infrastructure Enablement for {ibmpowerProductName} Virtual Server (Technology Preview)
```
Comment (manojnkumar) [inline release_notes/ocp-4-14-release-notes.adoc:265]: This is not new with this release.  We had TP of IPI in 4.13 as well.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 9]
```diff
@@ -203,9 +203,356 @@ For more information, see xref:../applications/creating_applications/creating-ap
...
+* Installing on a single node
+//* Hosted control planes (Technology Preview)
```
Comment (manojnkumar) [inline release_notes/ocp-4-14-release-notes.adoc:267]: Hosted Control Planes is in TP (is it being commented out)?

File: release_notes/ocp-4-14-release-notes.adoc [hunk 10]
```diff
@@ -203,9 +203,356 @@ For more information, see xref:../applications/creating_applications/creating-ap
...
+==== {ibmzProductName} and {linuxoneProductName} notable enhancements
+
+Starting with {product-title} 4.14, Extended Update Support (EUS) is extended to the {ibmzProductName} platform.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
+
+The {ibmzProductName} and {linuxoneProductName} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.
+
+This release introduces support for the following features on {ibmzProductName} and {linuxoneProductName}:
+
+* Assisted Installer with z/VM
+* Installing on a single node
```
Comment (manojnkumar) [inline release_notes/ocp-4-14-release-notes.adoc:228]: Add reference to storage?

Installing on a single node.  Persistent volumes for single node clusters can be provided by the LVM Operator.

File: release_notes/ocp-4-14-release-notes.adoc (hunk 9 above)
Comment (SNiemann15) [reply release_notes/ocp-4-14-release-notes.adoc:267]: Yes it has to be commented out. We will reintroduce it after HCP GA mid November through CM

File: release_notes/ocp-4-14-release-notes.adoc (hunk 8 above)
Comment (alishaIBM) [reply release_notes/ocp-4-14-release-notes.adoc]: Agree. Updated the PR.

File: release_notes/ocp-4-14-release-notes.adoc (hunk 10 above)
Comment (SNiemann15) [reply release_notes/ocp-4-14-release-notes.adoc:228]: No I wouldn't add any references in that list.

**Comment by SNiemann15** [comment]
/label peer-review-needed

India has two public holidays. Thereore adding the label for Alisha.  
QE review for IBM Z is done. For IBM Power I think too, only the ack from @manojnkumar missing.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 11]
```diff
@@ -203,9 +203,355 @@ For more information, see xref:../applications/creating_applications/creating-ap
...
+
+[IMPORTANT]
+====
+Compute nodes must run {op-system-first}
+====
+
+[discrete]
+==== {ibmzProductName} and {linuxoneProductName} notable enhancements
+
+Starting with {product-title} 4.14, Extended Update Support (EUS) is extended to the {ibmzProductName} platform.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
```
Comment (kelbrown20) [inline release_notes/ocp-4-14-release-notes.adoc:221]:
```suggestion
Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to the {ibmzProductName} platform.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 12]
```diff
@@ -203,9 +203,355 @@ For more information, see xref:../applications/creating_applications/creating-ap
...
+{product-title} now supports configuring {op-system-first} nodes for IBM Secure Execution on {ibmzProductName} and {linuxoneProductName} (s390x architecture).
+
+For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_z/installing-ibm-z-kvm.html#installing-rhcos-using-ibm-secure-execution_installing-ibm-z-kvm[Installing {op-system} using IBM Secure Execution]
+
 [id="ocp-4-14-ibm-power"]
 === {ibmpowerProductName}
 
+With this release, {ibmpowerProductName} is now compatible with {product-title} {product-version}. For installation instructions, see the following documentation:
```
Comment (kelbrown20) [inline release_notes/ocp-4-14-release-notes.adoc:245]:
```suggestion
{ibmpowerProductName} is now compatible with {product-title} {product-version}. For installation instructions, see the following documentation:
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 13]
```diff
@@ -203,9 +203,355 @@ For more information, see xref:../applications/creating_applications/creating-ap
...
+
+[IMPORTANT]
+====
+Compute nodes must run {op-system-first}
+====
+
+[discrete]
+==== {ibmpowerProductName} notable enhancements
+
+Starting with {product-title} 4.14, Extended Update Support (EUS) is extended to the {ibmpowerProductName} platform.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
```
Comment (kelbrown20) [inline release_notes/ocp-4-14-release-notes.adoc:258]:
```suggestion
Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to the {ibmpowerProductName} platform.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 14]
```diff
@@ -203,9 +203,355 @@ For more information, see xref:../applications/creating_applications/creating-ap
...
+
+The {ibmpowerProductName} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.
```
Comment (kelbrown20) [inline release_notes/ocp-4-14-release-notes.adoc:260]:
```suggestion
The {ibmpowerProductName} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components.
```

**Review by kelbrown20 — COMMENTED** [review]
Great work! Just some wording suggestions that are up to you!

**Comment by alishaIBM** [comment]
@manojnkumar can we go ahead with the merge review ? could you please confirm. Thanks.

**Comment by manojnkumar** [comment]
> @manojnkumar can we go ahead with the merge review ? could you please confirm. Thanks.

LGTM

File: release_notes/ocp-4-14-release-notes.adoc [hunk 15]
```diff
@@ -203,9 +203,355 @@ For more information, see xref:../applications/creating_applications/creating-ap
...
+This release introduces support for the following features on {ibmpowerProductName}:
+
+* {ibmpowerProductName} Virtual Server Block CSI Driver Operator (Technology Preview)
+* Installing on a single node
+//* Hosted control planes (Technology Preview)
+* Multi-architecture compute nodes
+* oc-mirror plugin
+
+[discrete]
+=== {ibmpowerProductName},{ibmzProductName}, and {linuxoneProductName} support matrix
```
Comment (jeana-redhat) [inline release_notes/ocp-4-14-release-notes.adoc:271]: Missing a space here
```suggestion
=== {ibmpowerProductName}, {ibmzProductName}, and {linuxoneProductName} support matrix
```

**Review by jeana-redhat — COMMENTED** [review]
Noticed a small typo while scanning through the output. This is otherwise looking ready to merge :+1:

File: release_notes/ocp-4-14-release-notes.adoc (hunk 15 above)
Comment (alishaIBM) [reply release_notes/ocp-4-14-release-notes.adoc]: @jeana-redhat thank you.
Updated the PR.

**Pull request merged**